apps/arena/lib/arena/game_socket_handler.ex

```diff
@@ -153,7 +153,7 @@ defmodule Arena.GameSocketHandler do
   end
 
   @impl true
-  def terminate(_reason, _req, %{game_finished: false} = state) do
+  def terminate(_reason, _req, %{game_finished: false, enable: true} = state) do
```

Comment:
Let's change this to check if the player is alive. That way if we change the `:enable` functionality we don't break this.

Reply:
Sounds good